deploy/vagrant/provision/roles/tachyon/tasks/config.yml

```diff
@@ -18,6 +18,14 @@
   script: zookeeper.sh
   when: tachyon_masters > 1
 
+- name: set journal folder as a shared folder in underfs for fault tolerance
+  script: set_journal_folder.sh
+  when: tachyon_masters > 1
```

Comment:
By default, single master will write its journal to $TACHYON_HOME/journal which is on local master node. In single master mode, the journal is for restarting the master, so storing to local host is more efficient I think.